components/telemetry-operator/internal/validation/plugin_validator_test.go

```diff
@@ -39,6 +39,7 @@ func TestValidateForbiddenFilters(t *testing.T) {
 	err := sut.Validate(logPipeline, logPipelines)
 
 	require.Error(t, err)
+	assert.Contains(t, err.Error(), "plugin 'grep' contains match condition. Match conditions are forbidden")
```

Comment:
Looks like the test before did not what the name suggests. Let's split it to two tests. One to validate that a LogPipeline that has a `match` expression is rejected and one that tests for the forbidden `lua` filter.